packages/abi-typegen/src/abi/types/EnumType.ts

```diff
@@ -75,14 +76,29 @@ export class EnumType extends AType implements IType {
     const attributeKey: 'inputLabel' | 'outputLabel' = `${target}Label`;
 
     const contents = enumComponents.map((component) => {
-      const { name, type: typeId } = component;
+      const { name, type: typeId, typeArguments } = component;
 
       if (typeId === 0) {
         return `${name}: []`;
       }
 
-      const { attributes } = findType({ types, typeId });
-      return `${name}: ${attributes[attributeKey]}`;
+      const type = findType({ types, typeId });
```

Comment:
This snippet is copied over from [`StructType`](https://github.com/FuelLabs/fuels-ts/blob/master/packages/abi-typegen/src/abi/types/StructType.ts#L55).